openedx/core/djangoapps/catalog/utils.py

```diff
@@ -40,18 +43,21 @@ def get_programs(uuid=None, type=None):  # pylint: disable=redefined-builtin
             return []
 
         api = create_catalog_api_client(user, catalog_integration)
+        types_param = ','.join(types) if types and isinstance(types, list) else None
```

Comment:
This is unnecessarily defensive, just go ahead and `','.join(types) if types else None`. Duck typing!